SkillMatrix.Domain/Languages/Language.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SkillMatrix.Domain.Languages
+{
+    public class Language
+    {
+        public long Id { get; set; }
+        public string Code { get; set; }
+        public string Name { get; set; }
+        public string NativeName { get; set; }
+
+        public virtual ICollection<Users.User> Users { get; set; }
```

Comment:
Maybe it would be better to import the Users namespace and put it here directly
```suggestion
        public virtual ICollection<User> Users { get; set; }
```